recipes/hydra-core/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "hydra-core" %}
+{% set version = "0.11.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebookresearch/hydra/archive/{{ version }}.tar.gz
+  sha256: 6d5b15297f486cf0a36fc7f5c8640004896ae3b878e8d236872fc0d74e823d4d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - omegaconf <=2.0
```

Comment:
Please make it as [specific](https://github.com/facebookresearch/hydra/blob/0.11_branch/setup.py#L107) as Hydra.
The intention behind that is to limit to OmegaConf 1.4.*.